src/content/docs/en/guides/astro-db.mdx

```diff
@@ -654,6 +655,49 @@ When you're ready to deploy, see our [Deploy with a Studio Connection guide](#de
 
 </ReadMore>
 
+## LibSQL
+
+<p><Since pkg="@astrojs/db" v="0.14.0" /></p>
+
+Alternatively to Astro Studio, Astro DB can connect to any LibSQL server. You can use an offering from any platform that exposes the Hrana protocol of the server or host it yourself.
+
+To connecto Astro DB to a LibSQL instead of Studio, set the following environment variables:
+- `ASTRO_DB_REMOTE_URL`: the connection URL to your LibSQL server
+- `ASTRO_DB_APP_TOKEN`: the auth token to your LibSQL server
+
+The environment variables have to be set during `astro build`, `astro db push --remote` and during runtime if your project use Astro DB on SSR pages.
```

Comment:
Can we explain or give an example of what someone needs to do to make sure these are set at the correct time? It's one thing to say they "have to be set during..", but it's helpful to show what they will need to do to make that happen. If simply setting the variables were enough, then this sentence wouldn't be necessary, right? So what else do they have to make sure they do?

Reply:
Setting the environment variables is enough, but they have to do it in all places being used. So, for example, when deploying to Vercel you need the environment variable set locally when you run `astro db push --remote` to apply the schemas _and_ on Vercel to connect to the DB.

Using Studio, the token only needs to be set on Vercel since the local run of `astro db push --remote` will use the user token.

Reply:
So, would this be added to the command as a flag when run? Or just added in the appropriate places (one and done, like in a `.env` file or in a dashboard somewhere? I think this is what I'm getting at. 

"Set when you run" could mean "pre-set, and available somewhere" or something you need to provide each time.

Reply:
I made the examples show the `.env` file so people put it there. There is multiple ways to do it along with security considerations, but that will depend on how their project and their connection to LibSQL is structured. I don't think there is much point going into details on Astro Docs vs following the docs of the hosting they are using